client/elife-theme/src/components/Button.js

```diff
@@ -0,0 +1,33 @@
+import { css } from 'styled-components'
+
+const extraSmall = css`
+  font-size: 11px;
+  line-height: calc(${props => props.theme.gridUnit} * 2);
+  padding: ${props => props.theme.gridUnit};
+`
+
+const small = css`
+  padding: ${props => props.theme.gridUnit};
+`
+
+const primary = css`
+  border: 0 none;
+`
+
+const reverse = css`
+  background: #262626;
```

Comment:
Shouldn't this be `colorText`?

Reply:
No, the reverse colors are not in the theme as far as I know. This is the reverse button in the invision file nick upload, which doesn't have any specification. I guess keeping this here for future use when we know all the details won't hurt.

Reply:
Might be worth double checking with Nick as I thought the point of the reverse colours is that the text colour becomes the background and the background becomes the foreground. `colorText` is `#212121` which is indistinguishably close this this.

Reply:
I do have an unanswered question about this in invision.